data/strings/strings.txt

```diff
@@ -38827,83 +38827,1970 @@
+    el = Ενημερώστε τους χάρτες για να λάβετε τις έγκυρες πληροφορίες σχετικά με τα αντικείμενα!
+    sk = Aktualizujte mapy, aby boli informácie o objektoch aktuálne!
+    sw = Sasisha ramani ili kuweka taarifa za vitu kuwa mpya!
+    zh-Hans = 请更新有关对象以获取实时信息的地图!
+    zh-Hant = 請更新有關對像以獲取實時信息的地圖!
+    fa = برای به روز نگه داشتن مناظر نقشه ها را به روز رسانی کنید!
+
+  [whats_new_auto_update_button_size]
+    en = Update (%s) and proceed
+    ru = Обновить (%s) и продолжить
+    ar = تحديث (%s) والمتابعة
+    cs = Aktualizujte (%s) a pokračujte
+    da = Opdater (%s) og fortsæt
+    nl = Bijwerken (%s) en verder gaan
+    fi = Päivitä (%s) ja jatka
+    fr = Mettre à jour (%s) et continuer
+    de = Erneuern (%s) und Fortsetzen
+    hu = Frissítés (%s) és folytatás
+    id = Perbarui (%s) dan lanjutkan
+    it = Aggiorna (%s) e continua
+    ja = アップデート(%s) して進む
+    ko = (%s) 업데이트 그리고 진행
+    nb = Oppdater (%s) og fortsett
+    pl = Zaktualizuj (%s) i kontynuuj
+    pt = Atualizar (%s) e continuar
+    pt-BR = Atualize (%s) e prossiga
+    ro = De actualizat (%s) și de continuat
+    es = Renovar (%s) y continuar
+    es-MX = Renovar (%s) y continuar
+    sv = Uppdatera (%s) och fortsätt
+    th = อัพเดต (%s) และกำลังอยู่ในขั้นตอนการทำงาน
+    tr = (%s) güncelle ve devam et
+    uk = Оновити (%s) та продовжити
+    vi = Cập nhận được (%s) và tiếp tục
+    el = Ενημέρωση (%s) και συνέχεια
+    sk = Aktualizovať (%s) a pokračovať
+    sw = Sasisha (%s) na endelea
+    zh-Hans = 更新（%s）并继续
+    zh-Hant = 更新（%s）並繼續
+    fa = به روز رسانی (%s) و ادامه
+
+  [dialog_pedestrian_route_is_long]
+    tags = ios,android
+    en = The pedestrian route is too long
+    ru = Пешеходный маршрут слишком длинный
+    ar = طريق المشاة طويل جدًا
+    cs = Trasa pro chodce je příliš dlouhá
+    da = Rute til fods er for langt
+    nl = De wandelroute is te lang
+    fi = Jalankulkureitti on liian pitkä
+    fr = La randonnée est trop long
+    de = Spazierweg ist zu lang
+    hu = A gyalogos útvonal túl hosszú
+    id = Rute pejalan kaki terlalu panjang
+    it = Percorso a piedi troppo lungo
+    ja = 歩行ルートが長過ぎます
+    ko = 보행자 루트가 너무 깁니다
+    nb = Fotgjenger-ruten er for lang
+    pl = Trasa piesza jest zbyt długą
+    pt = Percurso pedestre é muito longo
+    pt-BR = A rota para pedestres é muito longa
+    ro = Traseu pietonal prea lung
+    es = La ruta a pie es demasiado larga
+    es-MX = La ruta peatonal es demasiado larga
+    sv = Vandringsrutten är för lång
+    th = เส้นทางเดินเท้านั้นไกลเกินไป
+    tr = Yaya yolu çok uzun
+    uk = Пішохідний маршрут занадто довгий
+    vi = Tuyến đường đi bộ quá dài
+    el = Η πεζοπορική διαδρομή είναι πολύ μεγάλη
+    sk = Trasa pre chodcov je príliš dlhá
+    sw = Njia ya watembea kwa miguu ndefu sana
+    zh-Hans = 步行路线太长
+    zh-Hant = 步行路線太長
+    fa = مسیر عابر پیاده خیلی طولانی است
+
+  [whatsnew_toll_unpaved_title]
+    en = Lay out automobile routes considering all types of roads
```

Comment:
Lay out подразумевает, что пользователь сам проложит дороги. `Plan` И вообще из этого предложения следует, скорее, что пользователь может проложить маршрут по всем дорогам, а не то, что он может избежать каких-то дорог. 
Что-нибудь типа `Consider road type when planning automobile routes`, `Avoid certain road types in routing`

Reply:
получилось так: 

  [whatsnew_toll_unpaved_title]
    en = Consider road type when planning automobile routes
    ru = Прокладывайте автомаршруты с учетом типа дороги

 [whatsnew_toll_unpaved_message]
    en = Choose to avoid toll and unpaved roads and ferry crossings
    ru = Объезжайте платные и грунтовые дороги и паромные переправы